src/validateEmail.test.js

```diff
@@ -8,13 +8,57 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   it(`should return boolean`, () => {
-
+    expect(typeof validateEmail('test@mail.com')).toBe('boolean');
   });
 
   it(`should return 'true' for the valid email`, () => {
-    expect(validateEmail('test838@gmail.com.'))
+    expect(validateEmail('Alex.test-838@gmail.com'))
       .toBeTruthy();
   });
 
-  // write more tests here
+  it(`should return 'false' when the email not contains @`, () => {
+    expect(validateEmail('test838gmail.com'))
+      .toBe(false);
+  });
+
+  it(`should return 'false' for the email that begins with dot`, () => {
+    expect(validateEmail('.alextest-838@gmail.com'))
```

Comment:
Avoid breaking a string across multiple lines, as it can decrease code readability. Use a single line for the string instead.